demo/ContainerInteractor.swift

```diff
@@ -10,6 +10,7 @@ import Foundation
 import MapboxVision
 import MapboxVisionAR
 import MapboxVisionSafety
+import AVFoundation
```

Comment:
If it's used only for `CMSampleBuffer` it may be enough just import `CoreMedia`. What do you think?